crates/maybenot/src/time.rs

```diff
@@ -0,0 +1,61 @@
+use std::ops::AddAssign;
+
+/// Trait representing instants in time. Allows using maybenot frameworks with
```

Comment:
Please document, here or elsewhere, whether the instants returned from this type and/or provided to maybenot need to be monotonicly strictly increasing, monotonicly nondecreasing, or neither?

Reply:
I don't think that's a property of the trait. If we want to document that I think it should be on `Framework::trigger_events` which is the method where the user supplies instants into the library.

Reply:
Should be monotonically nondecreasing

Reply:
The trait has nothing to do with *creation* of times. The `now` method is not exposed on the trait. But I added this remark on the `trigger_events` method. What do you think?